src/grow.rs

```diff
@@ -47,3 +49,11 @@ pub enum GrowError {
     ZpoolSpawnError(std::io::Error),
     ZpoolOnlineFailed(ExitStatus),
 }
+
+impl fmt::Display for GrowError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "GrowError")
```

Comment:
Is this going to actually format the error, or just write GrowError?

Reply:
This will just write "GrowError"